app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -431,19 +431,22 @@ public void onResume() {
             showNearbyCardPermissionRationale();
         });
 
-        if (store.getBoolean("displayNearbyCardView", true)) {
-            checkPermissionsAndShowNearbyCardView();
-            if (nearbyNotificationCardView.cardViewVisibilityState == NearbyNotificationCardView.CardViewVisibilityState.READY) {
-                nearbyNotificationCardView.setVisibility(View.VISIBLE);
+        //notification cards should only be seen on contributions list, not in media details
+        if(mediaDetailPagerFragment == null){
```

Comment:
Sorry, more about style:
- Space after `//`
- First letter of comment in uppercase
- Space after `if` and before `{`

This is a general Java convention. You can see that convention being used in the following code lines actually :-)
Thanks!

Reply:
Fixed!